app/templates/main/date-filters.html

```diff
@@ -96,5 +96,6 @@
     <button type="submit"
             class="govuk-button govuk-button__browse-all-filters-form-apply-button"
             data-module="govuk-button">Apply filters</button>
-    <a href="{% if view_name == 'browse' %} {{ browse_url }} {% elif view_name == 'transferring_body' %} {{ transferring_body_url ~ transferring_body_id }} {% elif view_name == 'series' %} {{ series_url ~ series_id }} {% elif view_name == 'consignment' %} {{ consigmnent_url ~ consignment_id }} {% endif %}">clear filters</a>
+    <a class="govuk-link govuk-link--browse-all-filter"
+       href="{% if view_name == 'browse' %} {% if sorting_order %} {{ browse_url ~ sorting_order }} {% else %} {{ browse_url }} {% endif %} {% elif view_name == 'transferring_body' %} {% if sorting_order %} {{ transferring_body_url ~ transferring_body_id  ~ sorting_order }} {% else %} {{ transferring_body_url ~ transferring_body_id }} {% endif %} {% elif view_name == 'series' %} {% if sorting_order %} {{ series_url ~ series_id ~ sorting_order }} {% else %} {{ series_url ~ series_id }} {% endif %} {% elif view_name == 'consignment' %} {% if sorting_order %} {{ consigmnent_url ~ consignment_id ~ sorting_order }} {% else %} {{ consignment_url ~ consignment_id }} {% endif %} {% endif %}">Clear filters</a>
```

Comment:
worth expanding / adding an assertion to the clear filter e2e playwright test to show that the sorting order persists after clearing? I believe you already have the test for clearing so just need an extra assertion to show the order persists

Reply:
updated clear filter test cases as requested